tensorflow_io/core/python/experimental/serial_ops.py

```diff
@@ -0,0 +1,192 @@
+# Copyright 2020 The TensorFlow Authors. All Rights Reserved.
```

Comment:
The year should be 2021. Applicable to other newly created files as well.

Reply:
Done. I didn't change it in the older file I modified (tfio.core.python.api.experimental.serialization.py) but I changed it in the two I created.